geonamescache/__init__.py

```diff
@@ -7,54 +7,62 @@
 
 import json
 import os
+from typing import Any, Dict, List, Mapping, Optional, Tuple, TypeVar
```

Comment:
⚠️ NOTE: I did not update the version number (see line 3 of this file) and I leave the new version number up to your discretion.

I would recommend this be treated as a patch, as the underlying functionality hasn't changed, although a minor update would make sense too given the addition of a dependency.